docs/topics/settings.rst

```diff
@@ -604,6 +604,26 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
+-----------------------
+
+Default: ``False``
+
+Whether or not to allow broken responses, where declared ``Content-Length``
+does not match content sent by the server. If ``False``, these responses
+trigger a ``ResponseFailed`` error. If ``True``, these responses are allowed
+and the flag ``partial`` is added.
+
+Optionally, this can be set per-request basis by using the
+:reqmeta:`download_allow_dataloss` Request.meta key.
+
+.. note::
+
+  This feature needs Twisted >= 11.1 and the use of the default HTTP/1.1
```

Comment:
@kmike Do you think can we remove the note there? I copied that from one setting above.